src/web_interface/components/analysis_routes.py

```diff
@@ -90,7 +84,12 @@ def _get_correct_template(self, selected_analysis: str, fw_object: Union[Firmwar
             return self._get_analysis_view(selected_analysis)
         return get_template_as_string('show_analysis.html')
 
-    def _start_single_file_analysis(self, uid):
+    @roles_accepted(*PRIVILEGES['submit_analysis'])
+    @AppRoute('/analysis/<uid>', POST)
+    @AppRoute('/analysis/<uid>/ro/<root_uid>', POST)
+    @AppRoute('/analysis/<uid>/<selected_analysis>', POST)
+    @AppRoute('/analysis/<uid>/<selected_analysis>/ro/<root_uid>', POST)
+    def _start_single_file_analysis(self, uid, selected_analysis=None, root_uid=None):
         if user_has_privilege(current_user, privilege='submit_analysis'):
```

Comment:
This should not be necessary anymore, does it?

Reply:
You are totally right, I didn't even notice that -> changed